login.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="SoMe - Social media app for my profile" />
+    <title>SoMe One - Login</title>
+    <link rel="stylesheet" href="/dist/css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav
+        class="navbar navbar-expand-lg navbar-dark bg-darkorange"
+        aria-label="Eighth navbar example"
+      >
+        <div class="container">
+          <a class="navbar-brand" href="index.html">SoMe One</a>
+          <button
+            class="navbar-toggler bg-darkgrey"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarsExample07"
+            aria-controls="navbarsExample07"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarsExample07">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="index.html"
+                  >Home</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="profile.html">Profile</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="login.html">Login</a>
+              </li>
+            </ul>
+            <form role="search">
+              <input
+                class="form-control"
+                type="search"
+                placeholder="Search"
+                aria-label="Search"
+              />
+            </form>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="">
+      <div class="row">
+        <div class="col-md-12 d-flex justify-content-center">
+          <div class="page-header pb-5">
+            <h1 class="title">My <span>profile page</span></h1>
+          </div>
+        </div>
+      </div>
+
+      <!-- Login form -->
+      <div class="px-2">
+        <div
+          class="w-80 border border-dark container-sm form-width bg-lightred rounded-2"
+        >
+          <div class="p-2">
+            <div class="mb-3">
+              <form>
```

Comment:
You are missing the form `action` attribute here. The form should take you to /profile.html once a user signs in.